autonomx/Generator.cpp

```diff
@@ -680,26 +686,28 @@ void Generator::applyInputRegion() {
 
 
         // write region activation onto lattice in rect area
-            for(int x = region->getRect().x(); x < xMax; x++) {
+        for(int x = region->getRect().x(); x < xMax; x++) {
             for(int y = region->getRect().y(); y < yMax; y++) {
-                QString wolframCA = "WolframCA";
-                QString gameOfLife = "GameOfLife";
                 double valueToWrite;
                 double inputValue;
-                if(this->getType() == wolframCA || this->getType() == gameOfLife) {
+                // rounds values to integers if generator type is WolframCA or GameOfLife
+                if(this->getType() == "WolframCA" || this->getType() == "GameOfLife") {
```

Comment:
Use constants for the type names, such as "GameOfList" or "WolframCA". (static const char*)

Reply:
Hey, sorry, I'm not sure I understand how to do what you're suggesting. Are you saying I shouldn't use the "this->getType()" or the QString? The linked thread on stackoverflow was confusing to me in relation to this.